suzieq/poller/worker/services/arpnd.py

```diff
@@ -64,6 +64,10 @@ def _clean_nxos_data(self, processed_data, _):
                 drop_indices.append(i)
                 continue
 
+            if 'state' not in entry:
+                # textfsm version handling
+                entry['state'] = 'reachable'
```

Comment:
I saw we are doing this also for other nosses, so I'm not complaining. It's just a curiosity.
Why can we assume `state = reachable` if we do not get any state?